DarkScript3/GUI.cs

```diff
@@ -550,15 +680,15 @@ private void LoadColors()
             string[] lines = File.ReadAllLines("colors.cfg");
             foreach (string line in lines)
             {
-                string[] split = line.Split('=');
+                string[] split = line.Split(new[] { '=' }, 2);
                 string prop = split[0];
                 string val = split[1];
-                cfg[prop] = val;
+                cfg[prop] = val.Trim();
```

Comment:
Why not trim the value?

Reply:
I suppose I am trimming it, but I can trim the property name as well. This is mainly just a possible compatibility issue between \r and \r\n that wouldn't actually happen in practice, but may as well be more lenient.